lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -582,6 +582,17 @@ module.exports = class MethodMappings {
 
           return value;
         },
+        async isElementPresent(webElementOrShadowRoot) {
+          let element = await this.getWebElement(webElementOrShadowRoot);
+          if (element){
+            return true;
+          } 
+
+          element=await this.getShadowRoot(webElementOrShadowRoot);
```

Comment:
This method is non-existent.

Whenever you propose some changes, please try to test them yourself first by using them in some Nightwatch example test (present in `examples/tests` directory, `ecosia.js` and `duckDuckGo.js` are good examples to work with).

I don't think the added `isPresent` command would work in its present state.